src/Server/Remote/NativeSsh.php

```diff
@@ -85,12 +85,17 @@ public function run($command)
 
         $sshCommand = 'ssh ' . implode(' ', $sshOptions) . ' ' . escapeshellarg($username . $hostname) . ' ' . escapeshellarg($command);
 
-        $process = new Process($sshCommand);
-        $process
-            ->setPty($serverConfig->getPty())
-            ->setTimeout(null)
-            ->setIdleTimeout(null)
-            ->mustRun();
+        try {
+            $process = new Process($sshCommand);
+            $process
+                ->setPty($serverConfig->getPty())
+                ->setTimeout(null)
+                ->setIdleTimeout(null)
+                ->mustRun();
+        } catch (ProcessFailedException $exception) {
+            $errorMessage = \Deployer\isVerbose() ? $exception->getMessage() : $process->getErrorOutput();
```

Comment:
@elfet I used it but then noted that there is one usage of Verbose in the file. Okay, I'll update the PR.